README.md

```diff
@@ -70,6 +70,23 @@ Liqo is a platform to enable dynamic and decentralized resource sharing across K
 
 This quickstart lets you try Liqo in a playground environment built by two clusters in [KinD](https://kind.sigs.k8s.io/).
 
+### Install liqoctl
+
+First, set the variables corresponding to your set-up:
+
+```
+OS=linux # possible values: linux,windows,darwin
+ARCH=amd64 # possible values: amd64,arm64 
+```
+
+Then, you should execute the following commands to install the latest version of liqoctl:
+
+```
+curl --fail -LSO "https://get.liqo.io/liqoctl-${OS}-${ARCH}" && \
+chmod +x "liqoctl-${OS}-${ARCH}" && \
+sudo mv "liqoctl-${OS}-${ARCH}" /usr/local/bin/liqoctl
+```
+
```

Comment:
You might add the same sentence of the doc, about downloading from the release page